src/transaction/log_applier.c

```diff
@@ -4678,8 +4714,17 @@ la_get_recdes (LOG_LSA * lsa, LOG_PAGE * pgptr, RECDES * recdes, unsigned int *r
       OR_PUT_INT (recdes->data, repid_and_flag_bits);
 
       la_make_room_for_mvcc_insid (recdes);
-    }
 
+      if (recdes->length > la_Info.maxslotted_reclength)
```

Comment:
With a short explanation.
```suggestion
      // It will be coverted to REC_BIGONE which contains all mvcc flags on. 
      if (recdes->length > la_Info.maxslotted_reclength)
```